sps20/content/venue/contents.lr

```diff
@@ -1,38 +1,51 @@
-title: Venue
+title: Location
 ---
-description: A guide to the venue
+description: A guide to the conference venue
 ---
 ordering: 1
 ---
 body:
 
-# Venue
-The Swiss Python Summit is held on the campus of the **[OST Eastern Switzerland University of Applied Sciences](https://www.ost.ch/)** (formerly HSR) in **[Rapperswil](https://www.instagram.com/visitrapperswil/)** at the **Aula** in **Building 4** (upstairs).
```

Comment:
I kinda liked the `visitrapperswil` link, but I don't mind that much.

Reply:
Ok, I will add it back

Reply:
BTW, how is the "Ask hotels for discount" going? Or should we just add a section "accommodation" with a few tips? and if so, can yo write it?